components/DAP/source/spi_switch.c

```diff
@@ -18,6 +18,7 @@
 #include "components/DAP/include/cmsis_compiler.h"
 #include "components/DAP/include/spi_switch.h"
 #include "components/DAP/include/gpio_common.h"
+#include "DAP_config.h"
```

Comment:
Is it necessary?

Reply:
I used the definitions like `PIN_SWDIO_MOSI` from `DAP_config.h`, you are right that IOMUX pins are fixed anyway, it will cause error if user change the pin from DAP_config.h. I will change that.